src/Serialization/Exception/SerializedClassDoesNotExist.php

```diff
@@ -0,0 +1,13 @@
+<?php namespace SmoothPhp\Serialization\Exception;
+
+use Exception;
+
+/**
+ * Class SerializedClassDoesNotExist
+ * @package SmoothPhp\Serialization\Exception
+ * @author jrdn hannah <jrdn@jrdnhannah.co.uk>
+ */
+final class SerializedClassDoesNotExist extends Exception
+{
+
+}
```

Comment:
fuck off is that an actual rule what is the point why  
![Wat](http://i.imgur.com/GxxiIKC.gif)